lib/osut/utils.rb

```diff
@@ -2689,10 +2698,11 @@ def poly(pts = nil, vx = false, uq = false, co = true, tt = false, sq = :no)
       # Using a transformation that is most likely not specific to pts. The
       # most probable reason to retain this option is when testing for polygon
       # intersections, unions, etc., operations that typically require that
-      # points remain nonetheless 'aligned'. This logs a warning if aligned
-      # points aren't @Z =0, before 'flattening'.
-      invalid("points (non-aligned)", mth, 1, WRN) unless xyz?(a, :z, 0)
-      a = flatten(a).to_a                          unless xyz?(a, :z, 0)
+      # points remain nonetheless 'aligned'. If re-activated, this logs a
+      # warning if aligned points aren't @Z =0, before 'flattening'.
+      #
+      #   invalid("points (non-aligned)", mth, 1, WRN) unless xyz?(a, :z, 0)
+      a = flatten(a).to_a unless xyz?(a, :z, 0)
```

Comment:
Maybe better as a DEBUG log, but muting this for now. Upcoming methods will frequently need to project polygons unto each other (without sharing same 3D plane).